### DISCUSSION TITLE
Release with custom CA support for Artifact repository

### DISCUSSION THREAD

**Discussion by ramanNarasimhan77** [argoproj/argo-workflows#11846] [Q&A] [answered]
@terrytangyuan 
We are looking for a release that contains https://github.com/argoproj/argo-workflows/pull/11161 which was merged last month to master.
(Commit: https://github.com/argoproj/argo-workflows/commit/7b80ce19e8afe6690aed5c2f3d6c123c812e468b)

Recent releases do not include this commit, so I thought of asking here for information. 

Thanks
Raman

**Comment by terrytangyuan** [comment, marked as answer]
It will be available in 3.5rc2: https://github.com/argoproj/argo-workflows/issues/11381

**Reply by ramanNarasimhan77** [reply]
Thanks a lot for the answer.
